src/main/webapp/js/build-messages-helper.js

```diff
@@ -3,12 +3,12 @@
 function buildMessageDiv(message){
  const subjectDiv = document.createElement('div');
  subjectDiv.classList.add("left-align");
- subjectDiv.innerText = "Subject: "; 
+ subjectDiv.innerText = "Subject: ";
  subjectDiv.appendChild(document.createTextNode(message.subject));
 
  const usernameDiv = document.createElement('div');
  usernameDiv.classList.add("left-align");
- usernameDiv.appendChild(document.createTextNode(message.user));
+ usernameDiv.appendChild(createLink('/user-page.html?user=' + message.user, message.user));
```

Comment:
This createLink() method is defined in navigation-loader.js, right? That's not a problem (and good to reuse the code!), but be aware that it might be fragile if we ever tried to build a page without the navigation.

Reply:
Ok good to note. When would we ever build the feed/user page without the nav bar though?